dev_tools/bash_scripts_test.py

```diff
@@ -317,6 +319,8 @@ def test_pytest_changed_files_branch_selection(tmpdir_factory):
         tmpdir_factory=tmpdir_factory,
         setup='mkdir alt\n'
         'cd alt\n'
+        'export GIT_CONFIG_GLOBAL=/dev/null\n'
+        'export GIT_CONFIG_SYSTEM=/dev/null\n'
```

Comment:
This can be removed, because the `setup` argument for the `run()` function above
is inserted after the exports on lines 60-61.

Reply:
Ah, foo. I didn't notice how `run` was being used. Serves me right for trying to do things too quickly. Thanks for noticing and explaining that.